src/components/sprint-view-table/tasks-table.tsx

```diff
@@ -69,51 +71,34 @@ const TaskTable = ({project, loggedInpersonId, filter}: Props) => {
           }
           return 0;
         } catch (error) {
-          console.error(`Error fetching time entries for task ${task.id}: ${error}`);
+          const message: string = strings.formatString(strings.sprintRequestError.fetchTasksError, task.id||0, error as string).toString();
+          setError(message);
           return 0;
         }
       }));
       setTimeEntries(fetchedTimeEntries);
+      setLoading(false);
     } catch (error) {
-      console.error(`Error fetching time entries: ${error}`);
-    }
-  };
-
-  /**
-   * Retrieve total time entries for a task
-   */
-  const getTotalTimeEntries = (task: Tasks) => {
-    if (timeEntries.length!==0) {
-      return timeEntries[tasks.indexOf(task)];
+      setError(`${strings.sprintRequestError.fetchTimeEntriesError} ${error}`);
+      setLoading(false);
```

Comment:
No need to repeat the setLoading(false) just have it after the catch rather than in both the try and the catch